torch/_inductor/codegen/simd.py

```diff
@@ -1403,8 +1403,9 @@ def codegen_template(
                 for node in [template_node, *epilogue_nodes]:
                     node.mark_run()
             partial_code = render()
-            for node in epilogue_nodes:
-                node.codegen(kernel.split_and_set_ranges(node.get_ranges()))
+            with kernel.set_subgraph_body("<STORE_OUTPUT>"):
+                for node in epilogue_nodes:
+                    node.codegen(kernel.split_and_set_ranges(node.get_ranges()))
```

Comment:
OOC, how did you figure this out in the end?

Reply:
It's just how the templating works 🤔 When you call `node.codegen` on the epilogue nodes, it implicitly writes to global context (including `.body` potentially). Previously in my testing it just wasn't writing to `V.kernel.body` in this code, but the stack trace was indicating that it did here.